features/in_onderzoek.feature

```diff
@@ -1070,7 +1067,7 @@ Functionaliteit: in onderzoek
       | naam                | waarde                                              |
       | type                | RaadpleegMetBurgerservicenummer                     |
       | burgerservicenummer | 000000577                                           |
-      | fields              | adresregel1,adresregel2,straat,korteNaam,woonplaats |
+      | fields              | adresregel1,adresregel2,officieleStraatnaam,korteStraatnaam,woonplaats |
```

Comment:
Dit scenario is niet correct. straat, locatiebeschrijving of land is niet gezet waardoor deze persoon geen verblijfplaats heeft

Reply:
straatnaam met . (en gemeentecode) toegevoegd.